sweagent/environment/swe_env.py

```diff
@@ -506,8 +575,9 @@ def close(self) -> None:
             raise
         except:
             self.logger.warning("Errors when exiting container", exc_info=True)
-        assert self.container is not None  # mypy
-        self.container.terminate()
+        if self.container:
```

Comment:
Calling `close` on a closed env should not fail, since we also use it as a general "clean up before that next step" mechanism.